starter-code/styles/base.css

```diff
@@ -1,4 +1,8 @@
-/* TODO: Add a rule for images, so they never exceed their container's width. */
+
+img {
+  height: 30vh;
+  width: 30vw;
+}
```

Comment:
This causes your images to warp unnecessarily. You should use `max-width: 100%;` instead.